mythtv/libs/libmythui/mythmainwindow.cpp

```diff
@@ -715,22 +715,18 @@ void MythMainWindow::Init(bool MayReInit)
 
     move(m_screenRect.topLeft());
 
-    if (m_painterWin)
+    if (m_painterWin || m_painter)
     {
-        m_oldPainterWin = m_painterWin;
-        m_painterWin = nullptr;
+        LOG(VB_GENERAL, LOG_INFO, "Destroying painter and painter window");
+        MythPainterWindow::DestroyPainters(m_painterWin, m_painter);
     }
 
-    if (m_painter)
+    QString warningmsg = MythPainterWindow::CreatePainters(this, m_painterWin, m_painter);
+    if (!warningmsg.isEmpty())
     {
-        m_oldPainter = m_painter;
-        m_painter = nullptr;
+        LOG(VB_GENERAL, LOG_WARNING, warningmsg);
     }
-
-    QString warningmsg;
-    if (!m_painter && !m_painterWin)
-        warningmsg = MythPainterWindow::CreatePainters(this, m_painterWin, m_painter);
-
+    
```

Comment:
White space, please remove

Reply:
Remove just the spaces or the blank line?

Reply:
diff incorrectly interpreted the braces as not being changed
it should show remove:

```
if (m_painter)
{
    m_oldPainter = m_painter;
    m_painter = nullptr;
}
```

modify line 730
remove 731 and 732
add

```
if (!warningmsg.isEmpty())
{
    LOG(VB_GENERAL, LOG_WARNING, warningmsg);
}
```

Is a single line if statement considered acceptable style?  i.e.:

```
if (!warningmsg.isEmpty()) LOG(VB_GENERAL, LOG_WARNING, warningmsg);
```

Reply:
Single line if statements are generally never used. A single statement on the next line is accepted by the code checkers but only if the statement fits on a single line. Concluding, the style with the { } brackets even for a single following statement is preferred.
There is information in the Wiki https://www.mythtv.org/wiki/Coding_Standards but the actual code checkers are much much more strict.

Reply:
> Remove just the spaces or the blank line?

The spaces is what git complains about when creating a patch of your pull request and then applying that patch.